trinity/config.py

```diff
@@ -457,7 +464,7 @@ def node_class(self) -> Type['Node']:
         from trinity.nodes.full import FullNode
         from trinity.nodes.light import LightNode
 
-        if self.is_full_mode:
+        if self.is_full_mode or self.is_fast_mode:
```

Comment:
@pipermerriam currently I'm making `SYNC_FULL` and `SYNC_FAST` to use the same implementation here. I believe the current `FULL` implementation (i.e. `from trinity.nodes.full import FullNode`) will all need to be renamed to `FAST` (i.e. `from trinity.nodes.fast import FastNode`)?

Reply:
>So in the case of `Fast` syncing strategy, which node types that it should be using?

The node type remains being `FullNode`. So in terms of node types there are only `FullNode` and `LightNode` so far.